packages/myst-cli/src/process/mdast.ts

```diff
@@ -136,13 +136,13 @@ export async function transformMdast(
     new WikiTransformer(),
     new GithubTransformer(),
     new RRIDTransformer(),
-    new DOITransformer(), // This also is picked up in the next transform
+    // new DOITransformer(), // This also is picked up in the next transform
```

Comment:
Removing this here actually seems ot have no effect. What is the role of this transformer?